core/src/test/java/org/testcontainers/junit/wait/strategy/AbstractWaitStrategyTest.java

```diff
@@ -124,9 +123,8 @@ protected void waitUntilReadyAndSucceed(GenericContainer<?> container) {
         // start() blocks until successful or timeout
         container.start();
 
-        assertTrue(
-            String.format("Expected container to be ready after timeout of %sms", WAIT_TIMEOUT_MILLIS),
-            ready.get()
-        );
+        assertThat(ready.get())
```

Comment:
I am 99.9% sure this would work too (AssertJ supports `AtomicBoolean` OOTB):
```suggestion
        assertThat(ready)
```